src/v/kafka/server/handlers/fetch.h

```diff
@@ -351,4 +356,18 @@ ss::future<read_result> read_from_ntp(
   ssx::semaphore& memory_sem,
   ssx::semaphore& memory_fetch_sem);
 
+/**
+ * Create a fetch plan with the simple fetch planner.
+ *
+ * Exposed for testing/benchmarking only.
+ */
+kafka::fetch_plan make_simple_fetch_plan(op_context& octx);
+
+read_result::memory_units_t reserve_memory_units(
+  ssx::semaphore& memory_sem,
+  ssx::semaphore& memory_fetch_sem,
+  const size_t max_bytes,
+  const bool obligatory_batch_read);
```

Comment:
I think this is accidentally nabbed from 47892b7a9b6dec8e5142de7d071d830624f3e6ca